test/wasm_test.cc

```diff
@@ -15,6 +15,7 @@
 #include "include/proxy-wasm/wasm.h"
 
 #include "gtest/gtest.h"
+#include <unordered_set>
```

Comment:
Style: move `<unordered_set>` before `gtest/gtest.h` and add an empty line between them.

Reply:
Done

Reply:
You need to keep external include (`gtest/gtest.h`) separately from local include (`test/utility.h`), i.e.
```
#include "include/proxy-wasm/wasm.h"

#include <unordered_set>

#include "gtest/gtest.h"

#include "test/utility.h"
```

Reply:
Oh. my mistake. Fixed.